src/server.rs

```diff
@@ -1231,6 +1231,7 @@ where
 
             let me = me.clone();
             let cache_write = async move {
+                let _ = &me;
```

Comment:
Is this necessary? The variable should be moved either way, since `me.stats` is used later on.

Reply:
https://github.com/paritytech/canvas/pull/107#discussion_r751169325 for details.
This change was auto-generated from `cargo fix --edition` specifically. 

It's something that could be refactored in this PR if you possibly wanted to tweak it?

Reply:
In this particular situation it's even better to disjointly borrow (and immediately clone) the field, since the outer struct is `Arc` and the inner is `Arc<RwLock<ServerStats>>` - we're mostly interested about updating the stats and these can live independently.